client/components/common/tce-core/Discussion/Comment.vue

```diff
@@ -1,81 +1,143 @@
 <template>
-  <li class="comment">
-    <v-avatar size="34" class="comment-avatar">
-      <img :src="author.imgUrl">
-    </v-avatar>
-    <div class="comment-body pl-3">
-      <div class="header">
-        <span class="author">
-          {{ author.fullName || author.email }}
-          <v-icon v-if="isEdited" size="16" class="ml-1 pb-1">
-            mdi-pencil-outline
-          </v-icon>
+  <div>
+    <div v-if="showUnseenSeparator" class="unseen-separator">
+      <v-divider class="my-3" />
+      <v-chip @click="showSeenMarker = !showSeenMarker" small outlined>
+        Unseen
+      </v-chip>
+      <div class="d-flex justify-center">
+        <transition name="slide-fade">
+          <v-btn
+            v-if="showSeenMarker"
+            @click="$emit('markSeen')"
+            color="teal"
+            text x-small
+            class="seen-marker">
+            <v-icon class="mr-1" x-small>mdi-check</v-icon>
+            Mark All as Seen
+          </v-btn>
+        </transition>
+      </div>
+    </div>
+    <li class="comment">
+      <v-avatar size="34" class="comment-avatar">
+        <img :src="author.imgUrl">
+      </v-avatar>
+      <div class="comment-body pl-3">
+        <div class="header">
+          <span class="author">
+            {{ author.fullName || author.email }}
+            <v-icon v-if="isEdited" size="16" class="ml-1 pb-1">
+              mdi-pencil-outline
+            </v-icon>
+          </span>
+        </div>
+        <text-editor
+          v-model="content"
+          :is-focused="isEditing"
+          :show-preview="!isEditing"
+          class="content" />
+        <span v-if="isEditing" class="float-right">
+          <v-btn @click="reset" text small>Cancel</v-btn>
+          <v-btn
+            @click="save"
+            :disabled="content && !content.trim()"
+            color="green"
+            text small>
+            <v-icon class="pr-1">mdi-check</v-icon> Save
+          </v-btn>
         </span>
+        <div v-else class="footer">
+          <v-tooltip right>
+            <template v-slot:activator="{ on }">
+              <span v-on="on">
+                <timeago
+                  :datetime="comment.createdAt"
+                  :auto-update="60"
+                  class="time" />
+              </span>
+            </template>
+            <span>{{ comment.createdAt | formatDate('M/D/YY h:mm A') }}</span>
+          </v-tooltip>
+          <v-tooltip left>
+            <template v-slot:activator="{ on }">
+              <v-btn
+                v-if="elementTag"
+                v-on="on"
+                @click="toggleElementDiscussion"
+                color="teal"
+                text x-small>
+                Element
```

Comment:
When viewed within the list, to me it looks like it relates to the element below. To me it feels like it should be part of header.

Reply:
I sent some new UI/UX proposal changes in offline discussion.